### ISSUE TITLE
Unable to login on our GitLab server since update to version 7.9.0

### ISSUE THREAD

**Issue by ofrogon** [gitlabhq/gitlabhq#9014]
I did the update of GitLab from the version 7.8.4 to 7.9.0. All users using LDAP (active directory) connexion were no more able to login.

In the Log file, this error appear each times we try to login using LDAP: `LDAP search error: Invalid DN Syntax`

The installation was done using the Omnibus package and I didn't modify the gitlab.rb file.

I presume it is a bug from the 7.9.0 version because it worked before and it worked again after I rolled-back to 7.8.4

**Comment by dersk** [comment]
Thanks... I installed GitLab for the first time couple of hours ago and got stuck for while configuring ldap-support.
Fortunately I used docker, so switching to an older version wasn´t that hard - now it works.

tl;dr:
Can confirm, somewhere between 7.8.4 and 7.9.0 the ldap-support (at least for AD) got broken.

**Comment by dirknilius** [comment]
Same problem occured on our installation :( Rolled back to 7.8 and everything works again.

**Comment by setiseta** [comment]
Same here, rolled back to 7.8.
is there a config change to do with ldap auth?
any ideas?

**Comment by Razer6** [comment]
Today (March 25th) GitLab 7.9.1 will be released with a fix for this.
